packages/next/src/server/app-render/app-render.tsx

```diff
@@ -375,6 +375,7 @@ async function generateDynamicRSCPayload(
     skipFlight: boolean
   }
 ): Promise<RSCPayload> {
+  ctx.requestStore.phase = 'render'
```

Comment:
i'm not 100% sure if this is the way to do it -- i can see an argument that this should be set independently, before this function runs, and be an invariant here. but this works (in particular for the case of going `action` -> `render`) and it's hard to do it differently for `after` (which is currently kinda in control of when its stuff runs without exposing much to the outside world), so this is the convention i've settled on